hazelcast/src/test/java/com/hazelcast/spi/impl/servicemanager/impl/ServiceManagerImplTest.java

```diff
@@ -46,8 +46,7 @@ public void setup() {
 
         HazelcastInstance hz = createHazelcastInstance(config);
         NodeEngineImpl nodeEngine = getNodeEngineImpl(hz);
-        serviceManager = new ServiceManagerImpl(nodeEngine);
-        serviceManager.start();
+        serviceManager = (ServiceManagerImpl) nodeEngine.getServiceManager();
```

Comment:
this is a boy-scout change - it fixes another exception being logged during a test. it's in its own commit, see the commit message for more details.